src/odemis/acq/stream/_base.py

```diff
@@ -444,7 +444,7 @@ def _duplicateAxis(self, axis_name, actuator):
         pos = actuator.position.value[axis_name]
 
         if hasattr(axis, "choices"):
-            return model.FloatEnumerated(pos, choices=axis.choices, unit=axis.unit)
+            return model.VAEnumerated(pos, choices=axis.choices, unit=axis.unit)
```

Comment:
Why did you have to use a `VAEnumerated` instead of a `FloatEnumerated`? Note that the `FloatEnumerated` class accepts integers (or at least it should!). I don't think we have any axis which is not a float. So it should be compatible with everything...

Reply:
So this is what caused the problem with the actuators because there was one axis which was an int. Changing this back fixed the issue. But maybe I need to determine if I should make an IntEnumerated?

Reply:
So this is what caused the problem with the actuators because there was one axis which was an int. Changing this back fixed the issue. But maybe I need to determine if I should make an IntEnumerated?

Reply:
So this is what caused the problem with the actuators because there was one axis which was an int. Changing this back fixed the issue. But maybe I need to determine if I should make an IntEnumerated?